src/native/managed/cdacreader/Microsoft.Diagnostics.DataContractReader.Contracts/Contracts/RuntimeTypeSystem_1.cs

```diff
@@ -127,7 +126,7 @@ private static uint ComputeToken(Target target, Data.MethodDesc desc, Data.Metho
             uint tokenRemainder = (uint)(desc.Flags3AndTokenRemainder & tokenRemainderMask);
             uint tokenRange = ((uint)(chunk.FlagsAndTokenRange & tokenRangeMask)) << tokenRemainderBitCount;
 
-            return 0x06000000 | tokenRange | tokenRemainder;
+            return (uint)Constants.CorTokenType.mdtMethodDef | tokenRange | tokenRemainder;
```

Comment:
There is a common macro in C for this. Would it make sense to create a helper for creating these things?

For example:
```csharp
internal static uint CreateMethodDef(uint tkParts)
{
    Debug.Assert(tkParts & 0xff000000 == 0); // Validate token type isn't set
    return (uint)Constants.CorTokenType.mdtMethodDef | tkParts;
}
```